roles/operator/README.rst

```diff
@@ -0,0 +1,79 @@
+Ansible role to configure the operator with all its dependencies.
+
+**Role Variables**
+
+.. zuul:rolevar:: operator_user
+   :default: dragon
+
+The operator user.
+
+.. zuul:rolevar:: operator_user_id
+   :default: 45000
+
+Id for the operator user.
+
+.. zuul:rolevar:: operator_group
+   :default: dragon
+
+The group to which the operator user will be added.
+
+.. zuul:rolevar:: operator_group_id
+   :default: 45000
+
+Id for the group.
+
+.. zuul:rolevar:: operator_shell
+   :default: /bin/bash
+
+The default shell for the operator.
+
+.. zuul:rolevar:: operator_authorized_keys
+   :default: []
+
+Will be used to set ssh authorized keys if needed.
+
+.. zuul:rolevar:: operator_password
+
+   ----------------------------------------------------------------------------------
+   * ATTENTION                                                                      *
+   * Use "mkpasswd --method=sha-512" to generate a password                         *
+   * Do not use this variable and write the password into this file in clean words. *
+   ----------------------------------------------------------------------------------
+
+.. zuul:rolevar:: operator_sudo_nopasswd
+   :default: true
+
+Settings for the sudoers file.
+For that the dragon user don't have to type in the password for sudo rights.
+
+
+.. zuul:rolevar:: operator_sudo_cmd_list
+   :default: ALL
+
+Commands that the user can use, if some destricted or only some are allowed
+you can define it in this variable.
+
+.. zuul:rolevar:: result_create_operator_user.home
+
+The path to the home directory from a user. It will be used for language settings
+or for creating the .ssh directory for example.
+
+Example: ``/ubuntu/home/.bashrc``
+
+
+**Debian Variables**
```

Comment:
Since there is only one variable, I would drop this and move the description of the different default values to the text.